packages/cozy-mespapiers-lib/src/components/ModelSteps/Edit/ContactEditItem.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import { models } from 'cozy-client'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import ListItemSecondaryAction from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemSecondaryAction'
+import Avatar from 'cozy-ui/transpiled/react/Avatar'
+import Checkbox from 'cozy-ui/transpiled/react/Checkbox'
+import Radio from 'cozy-ui/transpiled/react/Radios'
+
+import styles from './styles.styl'
+
+const { getDisplayName } = models.contact
+
+const ContactEditItem = ({
+  contactIdsSelected,
+  setContactIdsSelected,
+  isMultiple,
+  contact
+}) => {
+  const { t } = useI18n()
+
+  const onClickContactLine = val => {
+    const newValue = val?.target?.value || val
+
+    if (isMultiple) {
+      const newContactIdSelected = [...contactIdsSelected]
+      const find = newContactIdSelected.indexOf(newValue)
+
+      if (find > -1) newContactIdSelected.splice(find, 1)
+      else newContactIdSelected.push(newValue)
+
+      setContactIdsSelected(newContactIdSelected)
+    } else {
+      setContactIdsSelected([newValue])
+    }
+  }
+
+  return (
+    <ListItem
+      button
+      key={contact._id}
+      onClick={() => onClickContactLine(contact._id)}
+    >
+      <ListItemIcon>
+        <Avatar size="small" className={styles['ContactEditList-Avatar']} />
```

Comment:
`<Avatar size="small" style={{color: '', backgroundColor: ''}} />` ou éventuellement sortir le style dans un object pour éviter la reconstruction...

Reply:
Euh, pouce!

Ayant la config Stylus sur la lib et après nos anciens échanges, j'en déduis:
- Composant MUI => makeStyles
- Autres => Stylus (sur UI ou comme ici vu que la config Stylus est en place), sinon style JS

🤔 🤷 💥

Reply:
tu peux rajouter à la règle que ca dépend 🤣 
le style est dans la doc de cozy-ui, en fait Avatar devrait potentiellement pouvoir avoir ça en prop directe... l'autre argument c'est que tant qu'on a pas 36 lignes de css, le stylus sert à rien.

En fait idéalement il ne devrait rien y avoir, ni style, ni stylus. Et c'est plus facile de supprimer des choses pour espérons ne rien avoir si on crée juste quelques lignes de code dans le style plutôt qu'un fichier stylus... qui va potentiellement inciter à être complété...

y'a d'autres sujets sur cette PR, on peut passer ça en nit si tu veux

Reply:
D'ailleurs, c'est pas plutôt un Circle que tu veux plutôt qu'un Avatar ? Vu que tu n'affiches ni texte ni image, j'imagine que tu veux juste le rond ? https://github.com/cozy/cozy-ui/blob/master/react/Circle/index.jsx

Reply:
fuckin' defaultProps at the end of the file! grrr. Thanks :)